rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/S3BinaryFieldTransformRequest.java

```diff
@@ -0,0 +1,234 @@
+package com.gentics.mesh.core.rest.node.field;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.gentics.mesh.core.rest.common.RestModel;
+import com.gentics.mesh.core.rest.node.field.image.FocalPoint;
+import com.gentics.mesh.parameter.image.CropMode;
+import com.gentics.mesh.parameter.image.ImageRect;
+import com.gentics.mesh.parameter.image.ResizeMode;
+
+/**
+ * POJO for a binary field transform request
+ */
+public class S3BinaryFieldTransformRequest implements RestModel {
```

Comment:
Can't we reuse the existing BinaryFieldTransformRequest, at it looks like identical?
Need also your opinions on this @bernhardriegler , @yrucrem .

Reply:
Actually I agree with you @plyhun . When i started working on it, I planned there would be some change in the functionality, which didn't happen. I would opt for the usage of BinaryFieldTransformRequest.

Reply:
@plyhun good suggestion - @rhoxhaj please go ahead and use the existing BinaryFieldTransformRequest